nvidia-bert/README.md

```diff
@@ -112,15 +112,17 @@ Note that the datasets used for BERT pre-training need a large amount of disk sp
     * Docker
     * [NVIDIA docker toolkit](https://github.com/NVIDIA/nvidia-docker)
 
-2. Pull the ONNX Runtime training docker image
+2. Build the ONNX Runtime training docker image
 
     ```bash
-    docker image pull <TODO INSERT MCR DOCKER IMAGE PATH HERE >
+    cd nvidia-bert/docker
+    bash build.sh
```

Comment:
See my comment about the Dockerfile.
Point to [BUILD.md in master repo](https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#Training) to build from source.
Can revert to using `build.sh` after we fix the Dockerfile.

Reply:
But that will change the rest of the instructions right?